lib/modules/manager/bundler/update.ts

```diff
@@ -0,0 +1,44 @@
+import { logger } from '../../../logger';
+import { escapeRegExp, regEx } from '../../../util/regex';
+import type { UpdateDependencyConfig } from '../types';
+
+export function updateDependency({
```

Comment:
why do we need this function?

Reply:
Because if we are to handle quotes in the Bundler manager side, it is not a simple string replacement.

Say the versioning module returns `>= 2.2.0, < 3` for `>= 1.4.0, < 2`. Then the Bundler manager needs to replace `'>= 1.4.0', '< 2'` with `'>= 2.2.0', '< 3'`. This is not something `doAutoReplace` can do.